File: app/src/main/java/mariavv/fitnesspal/presentation/NavigateView.java
package mariavv.fitnesspal.presentation;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.SkipStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;

@StateStrategyType(SkipStrategy.class)
public interface NavigateView extends MvpView {
    void setNavigator();

    void removeNavigator();

    void hideBottomMenu();

    void showBottomMenu();
}

File: app/src/main/java/mariavv/fitnesspal/presentation/NavigateView.kt
package mariavv.fitnesspal.presentation

import com.arellomobile.mvp.MvpView
import com.arellomobile.mvp.viewstate.strategy.SkipStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType

@StateStrategyType(SkipStrategy::class)
interface NavigateView : MvpView {
    fun setNavigator()

    fun removeNavigator()

    fun hideBottomMenu()

    fun showBottomMenu()

    fun showHomeAsUp(enabled: Boolean)
}
